topologies/ff.nix

```diff
@@ -0,0 +1,79 @@
+{
+  legacyCoreNodes = [];
+
+  legacyRelayNodes = [];
+
+  byronProxies = [];
+
+  coreNodes = [
+    # backup OBFT centralized nodes
+    {
+      name = "c-a-1";
+      region = "eu-central-1";
+      producers = [ "c-b-1" "c-c-1" "e-a-1" ];
+      org = "IOHK";
+      nodeId = 1;
+    }
+    {
+      name = "c-b-1";
+      region = "ap-northeast-1";
+      producers = [ "c-c-1" "c-a-1" "e-b-1" ];
+      org = "IOHK";
+      nodeId = 2;
+    }
+    {
+      name = "c-c-1";
+      region = "ap-southeast-1";
+      producers = [ "c-a-1" "c-b-1" "e-c-1" ];
+      org = "IOHK";
+      nodeId = 3;
+    }
+  ];
+
+  relayNodes = [
+    # stake pools
```

Comment:
this stake pools node will require some keys, right?
Will this be addressed in a follow-up PR?